onnxscript/ir/serde.py

```diff
@@ -512,10 +519,12 @@ def _deserialize_graph(
     # Initialize the values dictionary for this graph scope with the inputs and initializers
     values: dict[str, _core.Value] = {v.name: v for v in inputs}  # type: ignore[misc]
     scoped_values.append(values)
+    initializer_values = []
     for initializer in initializers:
```

Comment:
nit: consider renaming `initializers` to `initializer_tensors`?

Reply:
Done